packages/codemirror/src/codemirror-ipythongfm.ts

```diff
@@ -1,11 +1,30 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
 
-import CodeMirror from 'codemirror';
+import { Mode } from './mode';
+
+Mode.registerModeInfo({
+  name: 'ipythongfm',
+  mime: 'text/x-ipythongfm',
+  load: () => {
+    return import('@codemirror/lang-markdown').then(m => m.markdown());
+  }
+});
+
+// TODO: add support for LaTeX
+// This should be coded as an extension for https://github.com/lezer-parser/markdown
+// see https://github.com/lezer-parser/markdown/blob/main/src/extension.ts for examples
+// Then this extension can be passed to the markdown function above:
+//
+// load: () => {
+//   return import('@codemirror/lang-markdown').then(m => m.markdown({extensions: latex}));
+// }
+
+/*import CodeMirror from 'codemirror';
 import 'codemirror/addon/mode/multiplex';
 import 'codemirror/mode/gfm/gfm';
 import 'codemirror/mode/stex/stex';
-
+*/
```

Comment:
Why not import other modes, including stex from `codemirror/legacy-modes`?

Reply:
These modes are imported to be combined into a new one thanks to `CodeMirror.multiplexingMode` which is not available anymore in CodeMirror 6. Modes should now be based on Lezer and I am not sure we can reuse the legacy multiplex addon.